src/main/java/org/dimdev/dimdoors/pockets/SchematicV2Handler.java

```diff
@@ -31,10 +27,9 @@
     private static final Logger LOGGER = LogManager.getLogger();
     private static final Gson GSON = new GsonBuilder().setLenient().setPrettyPrinting().create();
     private static final SchematicV2Handler INSTANCE = new SchematicV2Handler();
-    private final List<PocketTemplateV2> templates = Lists.newArrayList();
-    private final Multimap<String, PocketTemplateV2> templateMap = HashMultimap.create();
-    private final List<PocketType> pocketTypes = Lists.newArrayList();
-    private static final Random RANDOM = new Random(new Random().nextLong());
+    private final Map<Identifier, PocketTemplateV2> templates = Maps.newHashMap();
+    private final Map<String, WeightedList<VirtualPocket, PocketGenerationParameters>> templateMap = Maps.newHashMap(); //TODO: un-ugly-fy
+    private final List<PocketGroup> pocketTypes = Lists.newArrayList();
```

Comment:
pocketGroups

Reply:
do you mean renaming the fields? How about:
``templateMap`` -> ``weightedPocketGroups``
``pocketTypes`` -> ``pocketGroups``

Reply:
even better 😅